codespan-reporting/tests/snapshots/fizz_buzz__rich_no_color.snap

```diff
@@ -1,71 +1,72 @@
 ---
-created: "2019-06-09T12:57:51.853099Z"
+created: "2019-06-23T08:14:40.662424Z"
 creator: insta@0.8.1
-source: codespan-reporting/tests/emit.rs
+source: codespan-reporting/tests/term.rs
 expression: result
 ---
 error[E0308]: `case` clauses have incompatible types
 
    ┌── FizzBuzz.fun:8:12 ───
-   │ 
+   │
  8 │     _ _ => num
    │            ^^^ expected `String`, found `Nat`
-   │ 
+   │
    = expected type `String`
         found type `Nat`
 
    ┌── FizzBuzz.fun:4:13 ───
-   │ 
- 4 │ fizz₁ num = case (mod num 5) (mod num 3) of
- 5 │     0 0 => "FizzBuzz"
- 6 │     0 _ => "Fizz"
- 7 │     _ 0 => "Buzz"
- 8 │     _ _ => num
-   │             -------------- `case` clauses have incompatible types
-   │ 
+   │
+ 4 │   fizz₁ num = case (mod num 5) (mod num 3) of
+   │ ╭─────────────-
```

Comment:
Shouldn't `-` be an arrow of sorts? It is rather ambigous wheter this points up or down (or at all)

Reply:
Yeah, I mentioned this here: https://github.com/brendanzab/codespan/issues/77#issuecomment-504724596

I think rustc gets away with this because it does:

```
___-
```

Reply:
One alternative is using another box drawing thing for secondary underlines:

```
─────────┘
```
or
```
─────────╯
```

Reply:
Open to ideas though!

Reply:
Another idea:

```
─‾
```

Using OVERLINE (U+203E) - might be less well supported by mono fonts though :/
```

Reply:
Using another box drawing makes it clear at least! I am well versed in unicode drawing so I don't have a better suggestion.

Overline looks good as well but if it can't be relied on to be there it doesn't seem like a good idea.

Reply:
Here's what overbar looks like alongside the carets we use for primary underlines:

```
foo <$> bar
‾‾‾ ^^^ ‾‾‾
```
doesn't look too bad I guess 🤔

Reply:
Another option using `'`:

```
  fizz₁ num = case (mod num 5) (mod num 3) of
╭─────────────'
```
```
foo <$> bar
''' ^^^ '''
```
And another using `"`
```
  fizz₁ num = case (mod num 5) (mod num 3) of
╭─────────────"
```
```
foo <$> bar
""" ^^^ """
```

Reply:
Oooh, perhaps we could combine `-` and `'` for secondary labels:

```
  fizz₁ num = case (mod num 5) (mod num 3) of
╭─────────────'
```
```
foo <$> bar
--- ^^^ --- a label about `bar`
```

Reply:
Tried this out in https://github.com/brendanzab/codespan/pull/98/commits/46cf1255c38c09325bd1f40cc6b7119be70dba0c